charts/opentelemetry-operator/crds/crd-opentelemetrycollector.yaml

```diff
@@ -0,0 +1,1394 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+    annotations:
+        cert-manager.io/inject-ca-from: opentelemetry-operator-system/opentelemetry-operator-serving-cert
+        controller-gen.kubebuilder.io/version: v0.6.0-beta.0
+    name: opentelemetrycollectors.opentelemetry.io
+spec:
+    conversion:
+        strategy: Webhook
+        webhook:
+            clientConfig:
+                caBundle: Cg==
+                service:
+                    name: webhook-service
+                    namespace: opentelemetry-operator-system
```

Comment:
Hardcoded namespace. I suppose it'll be broken if user installs chart in a different namespace or uses `overrideNamespace` option. If we can't use templates in this yaml, we could just completely ignore namespace provided with `helm install`, define `opentelemetry-operator-system` namespace right in the chart instead.

Reply:
This makes sense. How about we require users to install the Operator into the desired namespace, just like many Helm charts do? For example, I will write in the README that "using this command to install the OTEL Operator".

Reply:
I don't understand the reason to provide `namespaceOverride` config option which, if used, breaks the functionality. Same for `--namespace` attribute. Why can't we write in README that the operator helm release will be installed cluster-wise in a particular non-configurable namespace? Otherwise this file should be a helm template - I suppose that can be done in the release helper go script with some grep magic applied

Reply:
I don't understand the reason to provide namespaceOverride config option which, if used, breaks the functionality. Same for --namespace attribute. Why can't we write in README that the operator helm release will be installed cluster-wise in a particular non-configurable namespace? Otherwise this file should be a helm template - It should be doable to replace opentelemetry-operator-system with {{ template "opentelemetry-operator.namespace" . }} in the release helper script that you added.

Reply:
Yes we can do that. Since the collector CRD uses the hard-coded namespace, it probably best for us to use a non-configurable namespace as well. I will a YAML file under the `templates` directory to create the namespace if not exist. And I will make it clear in the README.

Reply:
I will delete the `namespaceOverride` option in values.yaml.